lib/internal/modules/helpers.js

```diff
@@ -298,14 +299,64 @@ function getBuiltinModule(id) {
   return normalizedId ? require(normalizedId) : undefined;
 }
 
+/** @type {import('internal/util/types')} */
+let _TYPES = null;
+/**
+ * Lazily loads and returns the internal/util/types module.
+ */
+function lazyTypes() {
+  if (_TYPES !== null) { return _TYPES; }
+  return _TYPES = require('internal/util/types');
+}
+
+let DECODER = null;
+/**
+ * Asserts that the given body is a buffer source (either a string, array buffer, or typed array).
+ * Throws an error if the body is not a buffer source.
```

Comment:
I think this line is redundant since there's a `@throws` tag that says basically the same thing.